### ISSUE TITLE
Add /az command to search for strings embedded in code

### ISSUE THREAD

**Issue by trufae** [radareorg/radare2#20848]
```

│           0x1000c9ec0      a92d8fd2       mov x9, 0x796d             ; 'my'
│           0x1000c9ec4      a9ccadf2       movk x9, 0x6e65, lsl 16    ; 'en'
│           0x1000c9ec8      694ccef2       movk x9, 0x7263, lsl 32    ; 'cr'
│           0x1000c9ecc      298feef2       movk x9, 0x7479, lsl 48    ; 'yt'
│           0x1000c9ed0      2aed8dd2       mov x10, 0x6f69            ; 'io'
│           0x1000c9ed4      ca6dadf2       movk x10, 0x6b6e, lsl 16    ; 'nk'
│           0x1000c9ed8      aa2ccff2       movk x10, 0x7965, lsl 32    ; 'ey'
│           0x1000c9edc      0ac0fdf2       movk x10, 0xee00, lsl 48
```

**Comment by alex-bender** [comment]
where can I find commit which implements that?

**Comment by trufae** [comment]
implemented in ba395426665fcd9964229eebd30a13e2dae62641

improved in :

* bc229b8f77f42e3e122bfba2d7c116f8a5249b2e
* 44d1fcf05b9c379728fec65a34a3c2b6fbbcaef1
* 56a140d1b2d1033fc0f13d57e5441575099cc7c4
* 7dbd325321a7cae136c17e4a445c15d33f3502a8
* 83446e8325c3b0c4ca49a217c4a5f353bd503efc
